helper/helper_embark_general.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# EMBArk - The firmware security scanning environment
+#
+# Copyright 2020-2022 Siemens Energy AG
+# Copyright 2020-2022 Siemens AG
+#
+# EMBArk comes with ABSOLUTELY NO WARRANTY.
+#
+# EMBArk is licensed under MIT
+#
+# Author(s): Benedikt Kuehne
+
+# Description: Helper functions
+
+docker_image_rm(){
+  # removes image by name and version
+  # $1 name
+  # $2 version
+  local IMAGE_NAME_="$1"
```

Comment:
for init an empty var in your local function ${1:-} will do the job:

local IMAGE_NAME_="${1:-}"